### PR TITLE
mostly wifi fixes

### PR DESCRIPTION
**Pull request by joshua255**
changed ints to 4 bytes-how esp32s and processing defines them, unlike the two byte ints of most arduinos

### CODE REVIEW

File: ursa.ino [hunk 1]
```diff
@@ -447,16 +446,20 @@ void addByteToBuffer(byte msg, byte &pos) {  // add a byte to the tosendData arr
...
   } d;  // d is the union, d.b acceses the byte array, d.v acceses the int
   d.v = msg;  // put the value into the union as an int
   dataToSend[pos] = d.b[0];
   pos++;
   dataToSend[pos] = d.b[1];
   pos++;
+  dataToSend[pos] = d.b[2];
+  pos++;
+  dataToSend[pos] = d.b[3];
+  pos++;
```
Comment (andrewda) [inline ursa.ino]: It might be cleaner to do this in a `for` loop instead

File: ursa.ino [hunk 2]
```diff
@@ -7,8 +7,8 @@
 
 #define ROBOT_ID 0      // unique robot ID, sent to DS, and used to name wifi network
 #define MODEL_NO 0      // unique configuration of robot which can be used to identify additional features
-#define MAX_SPEED 4000  // max speed (in steps/sec) that the motors can run at
-#define MAX_TIP 33.3    // max angle in degrees the robot will attempt to recover from -- if passed, robot will disable
+int MAX_SPEED = 4000; // max speed (in steps/sec) that the motors can run at
+float MAX_TIP = 60;  // max angle in degrees the robot will attempt to recover from -- if passed, robot will disable
```
Comment (andrewda) [inline ursa.ino]: Probably would make sense to move these below the following `#define`

**Review by andrewda — APPROVED** [review]
🔥

File: ursa.ino [hunk 3]
```diff
@@ -402,7 +402,7 @@ void zeroMPU6050() {  // find how much offset each gyro axis has to zero out dri
       digitalWrite(LED_BUILTIN, i % 2);
       delay(25);
     }
-  } while (abs(rotationOffsetX) > 100 || abs(rotationOffsetY) > 100 || abs(rotationOffsetZ) > 100);
+  } while (abs(rotationOffsetX) > 500 || abs(rotationOffsetY) > 500 || abs(rotationOffsetZ) > 500);
```
Comment (andrewda) [inline ursa.ino]: might be good to make this a variable, e.g. `movementThreshold`

**Review by andrewda — CHANGES_REQUESTED** [review]

File: ursa.ino (hunk 3 above)
Comment (joshua255) [reply ursa.ino]: I don't plan to change these numbers again, should they still be a variable?

Comment (andrewda) [reply ursa.ino]: Probably, just for readability's sake.

Comment (joshua255) [reply ursa.ino]: what do you think of 
`while (max(max(abs(rotationOffsetX),abs(rotationOffsetY)),abs(rotationOffsetZ)) > 500);`
so 500 is only written once?

Comment (joshua255) [reply ursa.ino]: on second thought, that is less readable, I'm pushing a commit using variables

**Review by andrewda — APPROVED** [review]
LGTM

**Pull request merged**